vault/resource_raft_autopilot.go

```diff
@@ -0,0 +1,151 @@
+package vault
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/vault/api"
+)
+
+var autopilotPath = "sys/storage/raft/autopilot/configuration"
+var autopilotDefaults = map[string]interface{}{
+	"cleanup_dead_servers":               false,
+	"dead_server_last_contact_threshold": "24h0m0s",
+	"last_contact_threshold":             "10s",
+	"max_trailing_logs":                  1000,
+	"min_quorum":                         0,
```

Comment:
Could we change this to match the default value to 3 as [listed here](https://www.vaultproject.io/api-docs/system/storage/raftautopilot#min_quorum)?

Reply:
Updated ✅